src/bp/nlu/typings_v1.d.ts

```diff
@@ -51,18 +61,33 @@ export interface PatternEntityDefinition {
 
 export type EntityDefinition = ListEntityDefinition | PatternEntityDefinition
 
-export interface PredictInput extends Credentials {
-  utterances: string[]
-}
+/**
+ * done : when a training is complete
+ * training-pending : when a training was launched, but the training process is not started yet
+ * training: when a chatbot is currently training
+ * canceled: when a training was canceled
+ * errored: when an unhandled error occured during training
+ *
+ * If the training does not exist, API returns a 404
```

Comment:
This comment shouldn't be here, we are not in `http-typings`

Reply:
yep